### ISSUE TITLE
Check Navmii

### ISSUE THREAD

**Issue by danieldegroot2** [matkoniecz/illegal-use-of-OpenStreetMap#34]
- Wiki: https://wiki.openstreetmap.org/wiki/Navmii
- Website: https://www.navmii.com/
- iOS App Store: https://apps.apple.com/us/app/navmii-offline-gps/id391334793

Plenty of mentions of OpenStreetMap, but where is attribution on any of their products? (Images on their websites do not show any attribution anywhere) Has anyone checked, i.e. if in the iOS app it's in the bottom left/right?

**Comment by habi** [comment]
I just downloaded the newest version from the AppStore.

![image](https://user-images.githubusercontent.com/1651235/177275172-5b953d4c-106b-4e3f-8cbc-0ea823dca163.jpeg)

The data they let you download (for Switzerland) is from December 2020 (!).

![image](https://user-images.githubusercontent.com/1651235/177275441-9cf72214-292a-41e7-bb29-e8be244600b0.png)

The map view shows *no* attribution at all, I was looking everywhere in the app an could not find a mention of OSM anywhere.

![image](https://user-images.githubusercontent.com/1651235/177275757-a60b2260-633a-461f-84f5-acd01e1ca536.png)

My personal 'trap street' is not on the map though, even though I added it on September 29, 2020...

**Comment by danieldegroot2** [comment]
@habi Some buildings are also not loading correctly or not present..

Does it still have a [map feedback](https://wiki.openstreetmap.org/wiki/Navmii#bug_reporting) tool, btw?

**Comment by habi** [comment]
> Does it still have a map feedback tool, btw?

Didn’t see something like this.
The App also looks completely different than the screenshots on the OSM wiki.

**Comment by habi** [comment]
> Some buildings are also not loading correctly or not present..

Yes, [this one here](https://www.openstreetmap.org/way/56248490/) (the longest residential building in Bern btw) is not rendered, no matter how I pan/zoom the map.
